apps/E2E/src/LinkExperimental/specs/ExperimentalLink.spec.win.ts

```diff
@@ -22,20 +21,32 @@ describe('Link Testing Initialization', function () {
 });
 
 describe('Link Accessibility Testing', () => {
-  it('Link - Validate accessibilityRole is correct', async () => {
+  beforeAll(async () => {
     await ExperimentalLinkPageObject.scrollToTestElement();
+  });
 
-    await expect(await ExperimentalLinkPageObject.getAccessibilityRole()).toEqual(LINK_A11Y_ROLE);
-    await expect(await ExperimentalLinkPageObject.didAssertPopup()).toBeFalsy(ExperimentalLinkPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  it('Validate accessibilityRole is correct', async () => {
+    await expect(
+      await ExperimentalLinkPageObject.compareAttribute(
+        ExperimentalLinkPageObject._primaryComponent,
+        Attribute.AccessibilityRole,
+        LINK_A11Y_ROLE,
+      ),
+    ).toBeTrue();
+
+    await expect(await ExperimentalLinkPageObject.didAssertPopup()).toBeFalsy(ExperimentalLinkPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped u
```

Comment:
nit not sure this comment is needed but there's a typo for "up" (same down below)